apps/README.md

```diff
@@ -30,19 +30,24 @@ yarn start
 ### Seeing your development version of Apps in Dashboard
 
 1. To make your changes show up in dashboard, do the following after the first time you build apps:
-  - Set `use_my_apps: true` to your locals.yml config file.
```

Comment:
my editor reformatted some extra stuff in this file when i made changes, sorry 🤷🏻‍♀️